shaders/Membrane.gdshader

```diff
@@ -29,28 +27,41 @@ void vertex(){
         * wigglyNess * size;
 }
 
+float fresnel(float power, vec3 normal, vec3 view)
+{
+	return clamp(pow(((1.0 - clamp(dot(normal, view), 0.0, 1.0)) * 0.875 + 0.125) * 3.0, power), 0.0, 1.0);
+}
+
 void fragment(){
     vec4 albedo = texture(albedoTexture, UV);
-    
-    vec4 normalmap = texture(normalTexture, UV);
-    vec3 normals = vec3(normalmap.r * 2.0 - 1.0, normalmap.g * 2.0 - 1.0, 
-        normalmap.b * 2.0 - 1.0);
 
-    float y = normals.y;
-    normals.y = -normals.z;
-    normals.z = y;
+    vec4 normalmap = texture(normalTexture, UV);
+	NORMAL_MAP = normalmap.xyz;
```

Comment:
Also seems like there's more tabs in this file, but I'll just comment this second time.